### ISSUE TITLE
Not catching all mixed content issues

### ISSUE THREAD

**Issue by ashfame** [Veejay/krolla#1]
Hi @Veejay

Currently, the tool doesn't detect images that issue mix content warning but is not blocked. Also, resources loaded as iframe objects or inside `<object>` are not caught.

Simply, compare the errors+warnings raised in chrome for https://googlesamples.github.io/web-fundamentals/fundamentals/security/prevent-mixed-content/active-mixed-content.html and the ones caught by the tool.

The above page, have `<object>` commented out so if you host the page yourself and uncomment it, you will see the warning for that one as well.

**Comment by Veejay** [comment]
Yeah, it's a rough first jab at it. I'll try to make it more solid in the future, just haven't had the time to get at it.

Since this is weekend leisure for me, I can't commit to any precise date for that to happen though.
